src/data/hicdex.js

```diff
@@ -472,16 +474,66 @@ export async function fetchObjkts(ids) {
   return data.token
 }
 
+async function fetchSubjktNames(ad) {
+  const { errors, data } = await fetchGraphQL(
+    `query SubjktNames ($addresses: [String!]) {
+    holder(where: {address: {_in: $addresses}}) {
+      address
+      name
+    }
+  }
+  `,
+    'SubjktNames',
+    { addresses: [...ad] }
+  )
+  if (errors) {
+    console.error('Failed to fetch SUBJKTs')
+    console.error(errors)
+  }
+  return data.holder
+}
+
 export async function fetchObjktDetails(id) {
   const { errors, data } = await fetchGraphQL(query_objkt, 'objkt', { id })
   if (errors) {
     console.error(errors)
   }
 
-  console.log(errors, data)
-
   const result = data.token_by_pk
-  console.log(result)
+
+  // Fetch burn transfers
+  // we want to use SUBJKT to resolve aliases not tzkt.
+  const addressesToResolve = []
+  result.transfers = []
+  const endpoint = `${process.env.REACT_APP_TZKT_API}/v1/operations/transactions?target.eq=${HEN_CONTRACT_FA2}&parameter.[*].txs.[*].token_id=${data.token_by_pk.id}&parameter.[*].txs.[*].to_=${BURN_ADDRESS}&level.gte=${data.token_by_pk.level}&entrypoint=transfer&status=applied`
+  try {
+    const burn_operations = await (await fetch(endpoint)).json()
+    burn_operations.forEach((item) => {
+      if (addressesToResolve.indexOf(item.sender.address) === -1) {
+        addressesToResolve.push(item.sender.address)
+      }
+
+      result.transfers.push({
+        timestamp: item.timestamp,
+        ophash: item.hash,
+        sender: item.sender.address,
+        opid: item.id,
+        amount: item.parameter.value[0].txs[0].amount,
```

Comment:
the txs array could have multiple entries. the burn transfer is not necessarily the first element in the array.

Reply:
I'm pretty sure it is given the endpoint call

Reply:
@xat So I used lodash 😅 tell me if it's what you had in mind